source/developer-guides/build-system.rst

```diff
@@ -59,6 +59,36 @@ Final |SPN| image(s) should be generated under ``Outputs/<platform_name>`` direc
 
 See :ref:`build-tool` for more details.
 
+Build Details per Stage
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Slim Bootloader is built in stages and more information on each stage is given below.
+
+* **Stage 1A**:
+
+  * **Packaged As**: FD containing Stage1A FV, and FSP-T binary as a FILE
+  * **Stage 1A FV**:
+
+    * Contains ``ResetVector``, ``VerInfo``, ``FlashMap``, ``FitTable``, ``HashStore``, ``PEIPcdDataBase``, and Stage1A PEIM
+    * ResetVector needs to be present at ``0xFFFFFFF0`` which is the reset vector location for x86.
+      The processor will fetch and execute code from this address as it comes out of reset.
+    * The Stage1A module itself needs to be placed at an appropriate address so that the reset vector matches

```

Comment:
Use same text as from the boot flow documentation

Reply:
Updated to match boot flow doc